docs/pages/management/admin/troubleshooting.mdx

```diff
@@ -161,3 +161,29 @@ cluster, read our [Teleport Diagnostics](../../reference/metrics.mdx) guide.
 For additional sources of Teleport support, please see the
 [Teleport Support and Education Center](https://goteleport.com/support/).
 
+## Common Issues
+
+### `teleport.cluster.local`
+
+It is common to see references to `teleport.cluster.local` within logs and
+errors in Teleport. This is a special value that is used within Teleport for two
+purposes and seeing it within your logs is not necessarily an indication that
+anything is incorrect.
+
+Firstly, Teleport uses this value within certificates (as a DNS SAN) issued to
```

Comment:
Maybe explain what a SAN is? (link to https://en.wikipedia.org/wiki/Subject_Alternative_Name etc)